microsoft-edge/webview2/concepts/browserfeatures.md

```diff
@@ -0,0 +1,93 @@
+---
+description: Feature differences between Microsoft Edge and WebView2
+title: Feature differences between Microsoft Edge and WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/08/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, WebView2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Feature differences between Microsoft Edge and WebView2  
+
+WebView2 is based off of the Microsoft Edge browser. This enables developers to have the opportunity to extend features from the browser to WebView2-based apps, which can be very useful. However, WebView2 is not limited to browser-like apps. There are many scenarios in which the browser's features don't make sense or are unneccesary.    
+
+The purpose of this article is to provide the following information.  
+
+*   The current approach that the WebView2 team uses for browser features.  
+*   The status of features and the reasoning behind the state.  
+*   The ability for features to be turned on \(or turned off\).  
+
+## Guiding Principles  
+
+Browser features within the context of WebView2 should adheres to the following guidelines for most WebView2 features.  
```

Comment:
How do we define browser features and what get included/excluded here?

Reply:
Right makes sense, I guess the idea was if any browser feature differs in Wv2 from edge, it should be listed here. I will add shopping & collections. I figure there might be alot more I'm missing and we should add it to this running doc later.